scripts/vespa_local/vespa_local.py

```diff
@@ -587,6 +599,15 @@ def deploy_config(args):
     os.system(f'vespa deploy "{here}"')
 
 
+def generate_application_package(args):
+    # Create instance of VespaLocal
+    # vespa_local_instance is used for starting vespa & generating application package.
+    vespa_local_instance = VespaLocalSingleNode()
+    # Generate the application package
+    file_path = vespa_local_instance.generate_application_package(zip=args.zip)
+    print(f"Application package generated at {file_path}")
```

Comment:
We can have a function `generate_and_deploy_application_package` that 
1. calls `generate_application_package` (with shards and replicas argument)
2. calls `deploy_application_package`
3. cleans up